osc-wrapper.py

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env python
-
-# this wrapper exists so it can be put into /usr/bin, but still allows the
-# python module to be called within the source directory during development
```

Comment:
Removing the wrapper is no-go.
We'll use entry point in rpm, but this wrapper script is used by many people that develop osc.
Please keep it as a thin wrapper calling babysitter.main()

Reply:
Ehm. Why? What’s missing from now updated PR?

Reply:
> Ehm. Why? What’s missing from now updated PR?

The whole osc-wrapper.py is missing.
I know that you can always run `setup.py develop` to achieve the same functionality, but that really doesn't always work for us. Please keep the osc-wrapper.py file around.